pkg/telemetry/metadata.go

```diff
@@ -61,12 +61,10 @@ func turnOffPackageLogging() {
 
 func getPkgEnv() string {
 	pkg := os.Getenv("PROMSCALE_PKG")
-	switch pkg {
-	case "deb", "rpm", "apk":
-		return pkg
-	default:
-		return "unknown"
+	if pkg == "" {
+		pkg = "unknown"
```

Comment:
Why not just include `docker` here like it was previously set? The reason I say this is because users can change the env variable and add random values which could potentially leak into our telemetry and cause issues for us.

I know it sounds like a long shot but users can, for instance, put an email address in and accuse us of breaking GDPR which would cause legal issues.

Reply:
Users can easily add email addresses in the telemetry they send anyway, so I don't know how this would be different. I believe we are already taking values from environment variables with tobs.

I don't think that's the right way to claim GDPR compliance. Our code is available, so anybody can check the code and see what it does and it will become apparent the data is sent by the user.

The problem with limiting this is that we need to change the code every time we want to add a new package which I don't think we need.

Reply:
If you think that it won't be an issue, then I'm OK with the change. I'll resolve this and approve the PR.